packages/ember-simple-auth-oauth2/lib/simple-auth-oauth2/authenticators/oauth2.js

```diff
@@ -218,13 +231,15 @@ export default Base.extend({
     @protected
   */
   makeRequest: function(url, data) {
-    return Ember.$.ajax({
-      url:         url,
-      type:        'POST',
-      data:        data,
-      dataType:    'json',
-      contentType: 'application/x-www-form-urlencoded'
-    });
+    var _this = this;
+    var options = {url: url, type: 'POST', data: data, dataType: 'json', contentType: 'application/x-www-form-urlencoded'};
+
+    if(!Ember.isEmpty(_this.clientId)) {
+      var base64ClientId = btoa(_this.clientId + ":");
```

Comment:
Sorry was sick for a few days.  However I could not get the AMD modules to correctly load the base64.js from utils for some reason so I gave up and just adapted the function from the project to be a private method in oauth2.js.  Maybe if you can put me in the right direction on how to pull in the project correct, get it wrapped in a module and loaded properly.  Until then this is working.